carbonmark-api/src/routes/projects/get.utils.ts

```diff
@@ -165,7 +188,17 @@ const pickUpdatedAt = (data: ProjectData): string => {
   const listings = compact(data.marketplaceProjectData?.listings || []);
   const youngestListing = maxBy(listings, (l) => Number(l.updatedAt));
   // if project has listings, use that value first, because the pool `lastUpdate` value is less relevant for users
-  return youngestListing?.updatedAt ?? data.poolProjectData?.lastUpdate ?? "";
+  const timestamps: string[] = [];
+  data.poolProjectData?.carbonCredits.forEach((credit) => {
+    credit.poolBalances.forEach((poolBalance) => {
+      poolBalance.pool.dailySnapshots.forEach((snapshot) => {
+        timestamps.push(snapshot.lastUpdateTimestamp);
+      });
+    });
+  });
+
+  const mostRecentTimestamp = maxBy(timestamps);
```

Comment:
Can probably just use `max` here
```suggestion
  const mostRecentTimestamp = max(timestamps);
```